cassandra-executor/src/main/java/com/mesosphere/dcos/cassandra/executor/CassandraDaemonProcess.java

```diff
@@ -431,10 +431,17 @@ public void drain()
      * @throws ExecutionException   If execution is interrupted.
      * @throws IOException          If communication with Cassandra fails.
      */
-    public void upgradeTables()
+    public void upgradeSSTables(String keyspace, List<String> columnFamilies)
```

Comment:
yeah, my bad. In previous DCOS version, get probe method calls were in respective task classes.